Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1513,6 +1515,11 @@ def prepare_REPACK(self, sequence = None):
         self.scheduleSequence(sequence.split('.')[-1],'digi2repack_step')
         return
 
+    def prepare_RAWPRIME(self, sequence = None):
```

Comment:
was a new step really necessary? Doesn't REPACK sort of do the same by nature?

Reply:
IIUC the repacked flag simply prepares the reconstruction for a different raw data label.  It doesn't actually implement the change to the data processing itself, in that case the application of a modified strip zero suppression algorithm at the HLT.  In the present case we are actually implementing the new algorithm in the workflow the for testing, as well as adding an additional collection to the raw data.  The raw data format itself has not yet been modified, as indicated in the comments.  Of course in the 2022 PbPb data taking, the cluster approximation will already be applied to the data coming out of the HLT.  Here we are simply preparing a workflow to test this functionality on data that was already recorded.

Reply:
Sorry, I had missed that we already had a REPACK step in addition to the --repack flag, but now I see it in 140.55
So your suggestion would be to add a new sequence to that to that step such as REPACK:ApproxClusters

Reply:
either that or, since a modifier is in place, it can convert the default `REPACK` sequence

Reply:
Ok, I think I followed your suggestion.  Please have a look.   However, I'm not sure how to make this work for both the proposed data and MC relval workflows, since the former uses repacked data and the latter doesn't.  In the current version of the PR 161 works, but 140.58 doesn't because the following line in DigiToRaw_Repack_cff.py is commented out:
siStripDigis.ProductLabel = "rawDataRepacker"
Do you have a suggestion?

Reply:
Since I implemented a customise that does the job, I'll mark this as resolved, although I suspect you'll have a different solution in mind.